src/cpp/src/image_generation/stable_diffusion_3_pipeline.hpp

```diff
@@ -545,8 +556,7 @@ class StableDiffusion3Pipeline : public DiffusionPipeline {
 
     std::shared_ptr<CLIPTextModelWithProjection> m_clip_text_encoder_1;
     std::shared_ptr<CLIPTextModelWithProjection> m_clip_text_encoder_2;
-    // TODO:
-    // std::shared_ptr<T5EncoderModel> m_t5_encoder;
+    std::shared_ptr<T5EncoderModel> m_t5_text_encoder;
     std::shared_ptr<SD3Transformer2DModel> m_transformer;
     std::shared_ptr<AutoencoderKL> m_vae;
```

Comment:
maybe for safely we need to initialize all `shared_ptr`s to `nullptr` by default?